tests/routes/test_validation_api.py

```
@@ -68,19 +68,18 @@ def test_message_validations_api(
     )
     assert session_response.status_code == 200
 
-    # Test content too long
+    long_content = "a" * 50001
     response = client.post(
         f"/v2/workspaces/{test_workspace.name}/sessions/{session_id}/messages",
         json={
             "messages": [
-                {"content": "a" * 50001, "peer_id": test_peer.name, "metadata": {}}
+                {"content": long_content, "peer_id": test_peer.name, "metadata": {}}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)